djangoplicity/pages/tests/factories.py

```diff
@@ -0,0 +1,39 @@
+import factory
+from djangoplicity.pages.models import (
+    Page, PageGroup, Section, URL, EmbeddedPageKey, PageProxy)
+
+
+class SectionFactory(factory.Factory):
+    class Meta:
+        model = Section
+
+    name = 'TestingSection'
```

Comment:
I find this order confusing, I think the attributes overrides before the meta settings are more understandable the same as Django models and Rest framework serializers.
```suggestion
    name = 'TestingSection'
    
    class Meta:
        model = Section
```

Reply:
resolved in [02217f](https://github.com/djangoplicity/djangoplicity/pull/96/commits/02217f9e80daec1adb482b895956e0597d9dfd18)